reek.rb

```diff
@@ -0,0 +1,3 @@
+def foo a,b,c,d,e,f,g,h,i,j,k
```

Comment:
__[development][rubocop]__
* Missing frozen string literal comment.
* Use def with parentheses when there are parameters. [:ref](https://github.com/bbatsov/ruby-style-guide#method-parens)
* Space missing after comma. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Unused method argument - `b`. If it's necessary, use `_` or `_b` as an argument name to indicate that it won't be used. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)
* Unused method argument - `c`. If it's necessary, use `_` or `_c` as an argument name to indicate that it won't be used. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)
* Unused method argument - `d`. If it's necessary, use `_` or `_d` as an argument name to indicate that it won't be used. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)
* Unused method argument - `e`. If it's necessary, use `_` or `_e` as an argument name to indicate that it won't be used. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)
* Unused method argument - `f`. If it's necessary, use `_` or `_f` as an argument name to indicate that it won't be used. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)
* Unused method argument - `g`. If it's necessary, use `_` or `_g` as an argument name to indicate that it won't be used. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)
* Unused method argument - `h`. If it's necessary, use `_` or `_h` as an argument name to indicate that it won't be used. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)
* Unused method argument - `i`. If it's necessary, use `_` or `_i` as an argument name to indicate that it won't be used. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)
* Unused method argument - `j`. If it's necessary, use `_` or `_j` as an argument name to indicate that it won't be used. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)
* Unused method argument - `k`. If it's necessary, use `_` or `_k` as an argument name to indicate that it won't be used. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)

<!-- Comment by [SideCI](https://sideci.com) -->